praw/models/comment_forest.py

```diff
@@ -72,12 +73,19 @@ def __len__(self) -> int:
         return len(self._comments)
 
     def _insert_comment(self, comment):
-        assert comment.name not in self._submission._comments_by_id
+        if comment.name not in self._submission._comments_by_id:
```

Comment:
Can we make a new subclass of client exception for this?

Reply:
I will if you prefer.

Reply:
Yes please.